### PR TITLE
Fixes #22106 - Clean up the common components

### PR DESCRIPTION
**Pull request by sharvit**
Clean up the common components directory to only contain sub-directories.

Before
```bash
Alert/
charts/
DocumentationLink/
forms/
Icon/
__snapshots__/
LoaderStyles.css
MessageBoxStyles.css
Loader.js
Loader.test.js
MessageBox.js
MessageBox.test.js
```

After
```bash
Alert/
charts/
DocumentationLink/
forms/
Icon/
Loader/
MessageBox/
```

### CODE REVIEW

**Review by ohadlevy — CHANGES_REQUESTED** [review]
I think you've missed simple loader in webpack/assets/javascripts/react_app/components/common/forms/Actions.js ? (maybe others)?

File: webpack/assets/javascripts/react_app/components/common/Loader/Loader.test.js [hunk 1]
```diff
@@ -2,10 +2,10 @@ import toJson from 'enzyme-to-json';
 import { shallow, mount } from 'enzyme';
 import React from 'react';
 
-import { STATUS } from '../../constants';
-import Loader from './Loader';
+import { STATUS } from '../../../constants';
+import Loader from './index';
```
Comment (ohadlevy) [inline webpack/assets/javascripts/react_app/components/common/Loader/Loader.test.js]: nitpick: ```import Loader from './'``` also works

**Comment by sharvit** [comment]
@ohadlevy - since the path moved from `Loader.js` to `Loader/index.js` the imports don't need to change.
I have just verified it work and tested.

File: webpack/assets/javascripts/react_app/components/common/Loader/Loader.test.js (hunk 1 above)
Comment (sharvit) [reply webpack/assets/javascripts/react_app/components/common/Loader/Loader.test.js]: tbh, i'm not sure what i prefer, i think `./index` is more reasonable to understand.

**Review by waldenraines — APPROVED** [review]
Looks fine to me.

**Comment by waldenraines** [comment]
@ohadlevy do you still require any changes here?

**Comment by waldenraines** [comment]
[test katello]

**Comment by sharvit** [comment]
@ohadlevy @waldenraines 

I have updated the tests to satisfy codeclimate.
Any other issues?

**Comment by waldenraines** [comment]
> I have updated the tests to satisfy codeclimate.
> Any other issues?

@sharvit nothing more from my perspective.

**Review by ohadlevy — APPROVED** [review]

**Pull request merged**